fdbserver/KeyValueStoreRocksDB.actor.cpp

```diff
@@ -2342,6 +2353,7 @@ struct RocksDBKeyValueStore : IKeyValueStore {
 	Future<Void> openFuture;
 	std::unique_ptr<rocksdb::WriteBatch> writeBatch;
 	std::set<Key> keysSet;
+	std::set<Key> previousCommitKeysSet;
```

Comment:
Can you add a comment on why this is needed? Something like "keeping the keys of in-flight commit".

Reply:
Done.